<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget3</class>
 <widget class="QMainWindow" name="DockWidget3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>213</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>213</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>421</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>213</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>421</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>321</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>213</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>213</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>213</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>213</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>321</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>123</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>321</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>321</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_re">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
